src/v2/view-builder/views/idp/RedirectIdvView.js

```diff
@@ -1,6 +1,22 @@
-import { loc } from '@okta/courage';
+import { loc, View } from '@okta/courage';
 import { BaseIdPAuthenticatorBody, BaseIdpAuthenticatorView } from './BaseIdpAuthenticator';
 import { FORMS } from 'v2/ion/RemediationConstants';
+import hbs from '@okta/handlebars-inline-precompile';
+import AuthenticatorFooter from 'v2/view-builder/components/AuthenticatorFooter';
+
+const PrivacyTermsFooterView = View.extend({
+  tagName: 'p',
+  className: 'margin-top-30',
+  template: hbs`
+      <p class="margin-btm-5">{{i18n code="oie.idv.idp.description.termsOfUse" bundle="login" 
+        $1="<a href='https://withpersona.com/legal/terms-of-use' class='terms-of-use inline-link' 
+          target='_blank' rel='noopener noreferrer'>$1</a>"
+        $2="<a href='https://withpersona.com/legal/privacy-policy' class='privacy-policy inline-link'
```

Comment:
If `.terms-of-use` and `.privacy-policy` are only used for tests, let's change them to `data-se`